src/main/kotlin/br/com/creditas/projethom/controller/EmployeeController.kt

```diff
@@ -39,4 +44,17 @@ class EmployeeController(
     ): EmployeeResponse = employeeService.registerEmployee(newEmployeeRequest)
 
 
+    @PutMapping("/{id}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    fun putEmployee(
```

Comment:
Em alguns casos como o put, soa estrando utilizar o nome desse método no nome da função, sugiro reutilizar o nome do Service. 
```suggestion
    fun updateEmployee(
```

Reply:
Ok, quando juntar as 3 branchs, eu vou alterar os nomes das funções. Pq acho que ta assim em todas